adds stamen library to map plugin assets and alters to support https

jduss4 opened the pull request:
followed techgique's recommentation here https://github.com/stamen/maps.stamen.com/issues/75
and PR https://github.com/stamen/maps.stamen.com/pull/119

jduss4 committed:
adds stamen library to map plugin assets and alters to support https

followed techgique's recommentation here https://github.com/stamen/maps.stamen.com/issues/75
and PR https://github.com/stamen/maps.stamen.com/pull/119

jechols reviewed (approved):
This change makes me a very happy panda.  Thank you!

techgique committed:
Clarify use of most recently changed version

The repository from which the JS file is copied has newer commits, but
the JS file itself hasn't been modified since the commit we document
